### DISCUSSION TITLE
Last iteration for Unity 2020?

### DISCUSSION THREAD

**Discussion by NeocortexVT** [happy-turtle/oit-unity#44] [Q&A]
From what I gather, the OIT-unity shader was at one point made for Unity 2020.3, correct? As the title suggest, I was wondering what the last iteration of the OIT shader is, that is still tested in Unity 2020 (2020.3.48 to be precise).

**Comment by happy-turtle** [comment]
You can try v3.0.1 then. That should have an interface still compatible with URP for Unity 2020. Assuming I am getting your question right.

**Reply by NeocortexVT** [reply]
Thank you, I'll play around with it to see if I can get it to work. For the BiRP, I gather I need to use the OitImageEffectComponent? Will it work if it is applied to a GameObject, or does it need to be assigned to a camera?

**Reply by happy-turtle** [reply]
Yes, that component was for the built-in pipeline. It uses [OnRenderImage](https://docs.unity3d.com/6000.0/Documentation/ScriptReference/MonoBehaviour.OnRenderImage.html), Unity's documentation states that it needs to be attached to a camera.

**Reply by NeocortexVT** [reply]
Thanks for clarifying!
